test/orchard/inspect_test.clj

```diff
@@ -897,6 +897,21 @@
                           (:newline)))
                       (datafy-section rendered))))))))
 
+(deftest inspect-eduction-test
+  (testing "inspecting eduction shows its obejct fields"
+    (let [rendered (-> (eduction (range 10)) inspect render)]
+      (testing "renders the header section"
+        (is (match? '("Class"
+                      ": "
+                      (:value "clojure.core.Eduction" 0)
```

Comment:
Probably worth playing with `inspect/next-page` as other tests do

Reply:
It's interaction with paging is weird right now (pager shows like it's paged but it's not, it just shows Java fields). See my comment below.

Reply:
Actually, fixed that weird interaction too since it was ugly.